packages/minifiers/minifiers.js

```diff
@@ -133,3 +133,24 @@ CssTools = {
   }
 };
 
+var toOSPath = function (p) {
```

Comment:
These is dumb, maybe we should extract these to a separate package and then uniload it in files.js